es.js

```diff
@@ -173,7 +173,7 @@ settings.interface.dmgCol Color de daño
 settings.interface.critCol Color de daño crítico
 settings.interface.header Interface
 settings.interface.scale Escala de interfaz
-settings.interface.theme Game Theme
+settings.interface.theme Tema
```

Comment:
Tema del Juego

Reply:
Hey, i noticed your translations are a bit confusing, not really inclusive, so i'll just try and delete some of them to resume it aswell as make it easier to apply the changes.

Thanks a lot for your work

- Nightz